src/queries/query.ts

```diff
@@ -14,6 +15,14 @@ import { executeIdMapQuery } from './catalog/id-map-query/id-map-query-executor'
 import type { IdMapQuery } from './catalog/id-map-query/id-map-query-format';
 import { executeNormalizedAstQuery } from './catalog/normalized-ast-query/normalized-ast-query-executor';
 import type {	NormalizedAstQuery } from './catalog/normalized-ast-query/normalized-ast-query-format';
+import { bold, type OutputFormatter } from '../util/ansi';
+import { printAsMs } from '../util/time';
+import { asciiCallContext, summarizeIdsIfTooLong } from '../cli/repl/commands/repl-query';
+import type { PipelineOutput } from '../core/steps/pipeline/pipeline';
+import type { DEFAULT_DATAFLOW_PIPELINE } from '../core/steps/pipeline/default-pipelines';
+import { graphToMermaidUrl } from '../util/mermaid/dfg';
+import { normalizedAstToMermaidUrl } from '../util/mermaid/ast';
+import Joi from 'joi';
 
 export type Query = CallContextQuery | DataflowQuery | NormalizedAstQuery | IdMapQuery;
```

Comment:
Cold we please move this type down so it can be inferred from Supported Queries and not the other way around?